mlrun/config.py

```diff
@@ -94,6 +94,10 @@
     "datastore": {"async_source_mode": "disabled"},
     # default node selector to be applied to all functions - json string base64 encoded format
     "default_function_node_selector": "e30=",
+    # default priority class to be applied to functions running on k8s cluster
+    "default_function_priority_class": "",
+    # valid options for priority classes - separated by a comma
+    "valid_function_priority_class_names": "",
```

Comment:
So what's the decision ? what empty here means ? that you don't validate ? or that you validate and nothing is valid ?
Should be mentioned in a comment here